src/lib/cdev/nuttx/cdev_platform.cpp

```diff
@@ -134,6 +152,8 @@ cdev_ioctl(file_t *filp, int cmd, unsigned long arg)
 static int
 cdev_poll(file_t *filp, px4_pollfd_struct_t *fds, bool setup)
 {
+	CHECK_INODE(-ENODEV);
```

Comment:
This is up to you, but I think I'd rather do it without the macro to make it a bit easier to read. All of these methods are already poking into the `f_inode` anyway.

Reply:
Done